presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParserErrorHandling.java

```diff
@@ -117,7 +117,11 @@
                  "line 1:19: mismatched input 'x'. Expecting: '(', ',', 'CROSS', 'EXCEPT', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LEFT', 'LIMIT', 'NATURAL', 'ORDER', " +
                          "'RIGHT', 'TABLESAMPLE', 'UNION', 'WHERE', <EOF>"},
                 {"SELECT * FROM t WHERE EXISTS (",
-                 "line 1:31: mismatched input '<EOF>'. Expecting: <query>"}};
+                 "line 1:31: mismatched input '<EOF>'. Expecting: <query>"},
+                {"SELECT \"\" FROM t",
+                 "line 1:8: mismatched input '\"'. Expecting: '*', 'ALL', 'DISTINCT', <expression>, <identifier>"},
+                {"SELECT a FROM \"\"",
+                 "line 1:15: mismatched input '\"'. Expecting: '(', 'LATERAL', 'UNNEST', <identifier>"}};
```

Comment:
Would it be possible to produce something more meaningful?
We sometimes accept something invalid in grammar in order to produce a better exc msg. For example, we could allow `""` as an identifier in grammar and check this when converting to AST?

Reply:
I like this idea. We could do this in `AstBuilder`:
```java
if (identifier.isEmpty()) {
    throw new ParsingException("Zero-length delimited identifier not allowed", getLocation(context));
}
```
PostgreSQL error message:
```
# select "";
ERROR:  zero-length delimited identifier at or near """"
LINE 1: select "";
```